style.css

```diff
@@ -366,10 +366,62 @@ table.schedule {
   border-spacing: 3px;
 }
 
-table.schedule span {
+table.schedule th span {
   font-size: 67%;
 }
 
+span.lightning, span.plenary, span.breakout, span.panel {
+  display: block;
+  margin: 0 5px 0 0;
+  font-size: 67%;
```

Comment:
this looks good in the schedule, but for the legend, the font-size could be bumped up for legibility and some margins could be added I think.

<img width="1153" alt="screen shot 2016-10-11 at 1 12 33 pm" src="https://cloud.githubusercontent.com/assets/203725/19286901/67601718-8fb4-11e6-8bdb-55fe8aedfda3.png">

<img width="1154" alt="screen shot 2016-10-11 at 1 10 31 pm" src="https://cloud.githubusercontent.com/assets/203725/19286886/558eab62-8fb4-11e6-8c0f-c4da2ced918b.png">


here are the styles I used:

```css
dt.format {
  box-shadow: .3em .3em 0 rgba(0,0,0,.1);
  font-size: 1rem;
  font-weight: 600;
  margin: 0 5px 15px 0;
  padding: 8px 10px;
}
```